### ISSUE TITLE
Describe how the accessibility label for the <summary> / <details> anon control can be provided (and add examples)

### ISSUE THREAD

**Issue by travisleithead** [w3c/html#250]
Per the thread here: https://www.w3.org/Bugs/Public/show_bug.cgi?id=25140 (now resolved) I think we have an action item recorded in this isssue.

@stevefaulkner @zcorpan
Looks like the following proposal had traction at the time:

``` html
<details>
  <input id="anonymous-control">
  <summary id="summary-anonymous-control">
    <label for="summary-anonymous-control">label text</label>
  </summary>
  Some content 
</details>
```

It allows the label to provide a larger hit-target on platforms where this is not the default convention, and also provides a label for the anonymous control used by the UA to perform the toggle. We need to describe the prose for how this works, and provide at least one example.

**Comment by zcorpan** [comment]
Thank you for the ping. I raised this issue in whatwg/html also.

**Comment by chaals** [comment]
I wrote a [quick and dumb test](http://chaals.github.io/testcases/summary-label-manual.html) which suggests there is currently zero implementation of this.

**Comment by travisleithead** [comment]
Doesn't work in Chrome on Windows, and Edge doesn't support the elements yet. @chaals are you suggesting this feature be incubated? It sounds like it has merit as an accessibility improvement...

**Comment by chaals** [comment]
Yes, I think the feature has merit but isn't ready.

Probably only because there is no implementation though - it matches the platform pretty obviously I think.

**Comment by travisleithead** [comment]
HTML5.1 Bugzilla Issue Triage: Incubation needed

This bug constitutes a request for a new feature of HTML. The current guidelines [1], rather than track such requests as bugs or issues, please create a proposal outlining the desired behavior, or at least a sketch of what is wanted (much of which is probably contained in this bug), and start the discussion/proposal in the WICG [2]. As your idea gains interest and momentum, it may be brought back into HTML through the Intent to Migrate process [3].

[1] https://github.com/w3c/html#contributing-to-this-repository
[2] https://www.w3.org/community/wicg/
[3] https://wicg.github.io/admin/intent-to-migrate.html
